stdlib/sys.pyi

```diff
@@ -125,16 +233,18 @@ class _implementation:
     version: _version_info
     hexversion: int
     cache_tag: str
-    _multiarch: str
```

Comment:
I checked, and this class isn't a `structseq` class: it's a `types.SimpleNamespace`. The `_multiarch` attribute doesn't seem to exist at runtime, though.

Reply:
This seems like a good use case for a `__getattr__` method.

Reply:
Good shout.

Reply:
Ah, and we don't even have to add another entry to the allowlist, since stubtest just ignores this class. Joy!